v22.2/choose-a-deployment-option.md

```diff
@@ -52,6 +52,28 @@ Cockroach Labs offers three ways to deploy CockroachDB: two managed services&mda
         <li><b>Advanced features</b>: Yes. See <a href="enterprise-licensing.html">Enterprise Features</a>.</li>
       </ul></td>
   </tr>
+  <tr>
+      <td><ul>
+        <li>All workloads: lightweight and critical production.</li>
+        <li>Applications that may need to grow and scale over time.</li>
+        <li>Applications with current and future requirements to grow into new cloud regions to serve customers in new markets.</li>
+        <li>Applications that require real-time integration with other systems.</li>
+        <li>Applications that need to be PCI compliant.</li>
```

Comment:
This one might be ok to leave as "PCI compliant" since it is talking about the application as a whole.